hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -123,7 +123,7 @@
   private String primaryUserGroup;
   private static final String DATE_TIME_PATTERN = "E, dd MMM yyyy HH:mm:ss z";
   private static final String TOKEN_DATE_PATTERN = "yyyy-MM-dd'T'HH:mm:ss.SSSSSSS'Z'";
-  private static final String XMS_PROPERTIES_ENCODING = "ISO-8859-1";
+  static final String XMS_PROPERTIES_ENCODING = "ISO-8859-1";
```

Comment:
Let's keep this private. This information doesn't need to be exposed to the callers. Add helper methods to this class to encode/decode strings from AzureBlobFileSystem.

Reply:
Done in fd7dc90.